src/date-range-input/date-range-input.tsx

```diff
@@ -360,11 +389,23 @@ export const DateRangeInput = ({
                 .startOf("week")
                 .format("YYYY-MM-DD");
 
-            setIsTouch(true);
+            setIsStartDisabled(true);
+            setIsEndDisabled(true);
             setInitialCalendarDate(firstDayOfWeek);
         }
     };
 
+    const handleFixedRangeSelectionInputFocus = () => {
+        if (isFixedRangeSelection) {
+            setIsEndDisabled(true);
```

Comment:
should also set `initialCalendarDate` to `selectedStart` here

for the case when date range is prefilled, we'll want to jump to the month of the selection by default